src/app/(pages)/layout.tsx

```
@@ -0,0 +1,73 @@
+
+export default function PagesLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  const { currentEstablishment, fetchMenuItems, fetchEstablishments } =
+    useNavigation();
+  const pathname = usePathname();
+
+  useEffect(() => {
+    fetchMenuItems();
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)